### ISSUE TITLE
[bug]: Progress images and job status are not updated in WebUI (websockets issue?)

### ISSUE THREAD

**Issue by sd92-sk** [invoke-ai/InvokeAI#4899]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cpu

### What version did you experience this issue on?

3.3.0post2, 3.3.0post1, 3.0.2post1(fresh install)

### What happened?

Users on discord with latest version or older versions see, that when generating image, progress images are not showing, also WebUI doesn't inform that job has stopped already.
In console we see the steps progress, at the end summary with timings. But no change in webui.
User has to refresh the page (F5) to get what's rendered.

Issue happens on Windows and also Linux. Users are seeing this with recent 3.3.0post1, but I also tried installing older 3.0.2post1 on linux (different tshoot ongoing) and same thing started happening here, on older 3.0.2post1 install this wasn't happening.

From observations above I assume that some dependency used for websockets communication between browser and InvokeAI backend could have introduced this issue?

### Additional context

Due to other problems i'm at this moment testing with 3.0.2post1, after refreshing the page, I see that in console is following error message:

```
/home/user/invokeai/.venv/lib/python3.11/site-packages/invokeai/app/api/sockets.py:30: RuntimeWarning: coroutine 'AsyncServer.enter_room' was never awaited
  self.__sio.enter_room(sid, data["session"])
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```
This might look differently on newer versions.

While looking around for existing issues, https://github.com/invoke-ai/InvokeAI/issues/4707 looks similar but it's about not updating the webui while multiple images are generated, i.e. a batch job; or user didn't try with single images; according to comment below this should have been fixed.

### Contact Details

I'm on DC under name sd92sk

**Comment by sd92-sk** [comment]
[4899_websockets_pcaps.zip](https://github.com/invoke-ai/InvokeAI/files/12910936/4899_websockets_pcaps.zip)
I'm attaching what I see in wireshark. In console I've looked on
```
[2023-10-16 01:42:38,149]::[uvicorn.error]::INFO --> ('192.168.1.166', 62060) - "WebSocket /socket.io/?EIO=4&transport=websocket&sid=ACQqLFaICdvvLbMiAAAC" [accepted]
[2023-10-16 01:42:38,150]::[uvicorn.error]::INFO --> connection open
```
and filtered for that port.

**Comment by Millu** [comment]
This was caused by the 5.10.0 release of `python-socketio`. There's a fix coming in the updated release (3.3.0post3)
